test/index.js

```diff
@@ -450,6 +450,62 @@ describe('DiffPatcher', () => {
         expectFormat([1, 2, 3], [1], [removeOp('/2'), removeOp('/1')]);
       });
 
+      it('should move elements in the correct order and remove properties of elements', () => {
+
+        instance = new DiffPatcher({
+          objectHash(obj) {
+            if (obj && obj.id) {
+              return obj.id;
+            }
+          },
+        });
+
+        expectFormat(
+          {
+            'hl': [
+              { id: 1, paramOne: 'bla', paramTwo: 'hello' }, // 0
+              { id: 2, paramOne: 'ga' }, // 1
+            ],
+          },
+          { 'hl': [{ id: 2, paramOne: 'ga' }, 
+          { id: 1, paramOne: 'bla' }] 
+        },
+          [
+            moveOp('/hl/1', '/hl/0'),
+            removeOp('/hl/1/paramTwo'),
+          ]);
+      });
+
+      it.skip('should move elements in the correct order and remove properties of elements without move operation', () => {
```

Comment:
This is the test case for the `remove`/`add` variant

Reply:
Do we know if it breaks "differently"? Just want to make sure that our down-stream patches still work.